README.md

```diff
@@ -80,28 +78,61 @@ the `config.yaml` file and change the following line:
 ```yaml
 runlog: false
```

Comment:
I don't think it is true that the `runlog` is `false`, so we can remove this bit.

Reply:
In these configs, it is `false` currently (or should be)

Reply:
I think these should be part of a separate issue - see https://github.com/ACCESS-NRI/access-om3-wav-configs/issues/51